charts/hono/templates/hono-service-command-router/hono-service-command-router-secret.yaml

```diff
@@ -88,6 +88,7 @@ stringData:
         certPath: {{ .Values.adapters.commandAndControlSpec.certPath }}
         trustStorePath: {{ .Values.adapters.commandAndControlSpec.trustStorePath }}
         hostnameVerificationRequired: {{ .Values.adapters.commandAndControlSpec.hostnameVerificationRequired }}
+        useLegacyTraceContextFormat: {{ .Values.adapters.commandAndControlSpec.useLegacyTraceContextFormat }}
```

Comment:
Do we need/want to be able to set this property per component? IMHO it should be sufficient to set this at a global level, e.g. using something like `.Values.useLegacyTraceContextFormat`.

WDYT?

Reply:
I've added a top-level `useLegacyAmqpTraceContextFormat` property now.